### PR TITLE
Fix README files of inferable quantizers

### PR DESCRIPTION
**Pull request by reuvenperetz**

### CODE REVIEW

File: model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md [hunk 1]
```diff
@@ -1,58 +1,53 @@
...
-- `inputs`: input tensor to quantize
-
-The method returns the quantized tensor.
-
-You must implement the abstract method `__call__` in your subclass which inherits BaseKerasInferableQuantizer
+Several Keras inferable quantizers were implemented for activation quantization:
+```markdown
+ActivationPOTInferableQuantizer
+ActivationSymmetricInferableQuantizer
+ActivationUniformInferableQuantizer
```
Comment (eladc-git) [inline model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md:12]: Maybe here you can reference to the folder contains them

File: model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md [hunk 2]
```diff
@@ -1,58 +1,53 @@
...
+                                                                             channel_axis=-1,
+                                                                             signed=True)
+
+# Initialize a random input qo quantize
+input_tensor = tf.random.uniform(shape=(1, 3, 3, 3), minval=-100, maxval=100)
+# Quantize tensor
 quantized_tensor = quantizer(input_tensor)
 print(quantized_tensor)
-```
+assert tf.reduce_max(quantized_tensor) < 4, f'Quantized values should not contain values greater than maximal threshold'
```
Comment (eladc-git) [inline model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md:47]: Why the user need this assert?

Comment (reuvenperetz) [reply model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md:47]: It does not need it, but it helps to understand what we expect to be the output

Comment (eladc-git) [reply model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md:47]: So maybe expain what the expectation after this code section.

**Review by eladc-git — APPROVED** [review]

**Pull request merged**